analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ClearTextProtocolsAreSensitiveTest.cs

```diff
@@ -34,17 +34,26 @@ public class ClearTextProtocolsAreSensitiveTest
     {
         [TestMethod]
         public void ClearTextProtocolsAreSensitive() =>
-            Verifier.VerifyAnalyzer(@"TestCases\Hotspots\ClearTextProtocolsAreSensitive.cs",
-                                    new ClearTextProtocolsAreSensitive(AnalyzerConfiguration.AlwaysEnabled),
-                                    ParseOptionsHelper.FromCSharp8,
-                                    AdditionalReferences);
+            Verifier.VerifyAnalyzer(
+                @"TestCases\Hotspots\ClearTextProtocolsAreSensitive.cs",
+                new ClearTextProtocolsAreSensitive(AnalyzerConfiguration.AlwaysEnabled),
+                ParseOptionsHelper.FromCSharp8,
+                AdditionalReferences);
 
 #if NET5_0
```

Comment:
We usually use `#if NET` instead of this and I would replace also this occurrence to avoid surprises as we change the runtime to NET6